multiview/cotraining/base.py

```diff
@@ -60,15 +64,17 @@ def __init__(self,
     def fit(self, Xs, y):
         """
         A method to co-trained estimators to multiview data.
+
         Parameters
         ----------
-        Xs: list of array-likes
+        Xs: numpy.ndarray, list of array-likes
             - Xs shape: (n_views,)
             - Xs[i] shape: (n_samples, n_features_i)
         y : array, shape (n_samples,)
+
         Returns
         -------
-        y_pred : array-like (n_samples,)
+        self : object
```

Comment:
sklearn says "returns an instance of self." instead of "object". not necessarily necessary but just saying